samples/net/mqtt_publisher/src/main.c

```diff
@@ -61,15 +61,41 @@ struct mqtt_client_ctx {
 	void *publish_data;
 };
 
-/* This is the network context structure. */
-static struct net_context *net_ctx;
-
 /* The mqtt client struct */
 static struct mqtt_client_ctx client_ctx;
 
 /* This routine sets some basic properties for the network context variable */
-static int network_setup(struct net_context **net_ctx, const char *local_addr,
-			 const char *server_addr, u16_t server_port);
+static int network_setup(void);
+
+#if defined(CONFIG_MQTT_LIB_TLS)
+/* TLS */
+#define TLS_SNI_HOSTNAME "localhost"
+#define TLS_REQUEST_BUF_SIZE 1500
+#define TLS_PRIVATE_DATA "Zephyr TLS mqtt publisher"
+
+#if !defined(CONFIG_NET_APP_TLS_STACK_SIZE)
```

Comment:
The TLS stack size value is set by default so no need to set it here too. If the default stack size is too long, then its value can be tweaked in prj conf file. I noticed that I had that setup like this in echo-client too but I will need to change that.